rfc/0032-field-level-encryption.md

```diff
@@ -283,10 +283,11 @@ Where:
 | ------------- |:-------------:|
 |PREFIX | Prefix for temporary field that holds content of encrypted field. This should be configurable so that naming conflicts can be easily resolved at the application level. The default is “crypt”.|
 |FIELDNAME |The original name of the field that contained the data to be encrypted |
+|BASE64() | Standard Base64 encoding. The results MUST include padding if applicable. Padding is required because the encoded results are included in signature calculation, which must be consistent across implementations.|
```

Comment:
I think we elected to not support padding sometime back?

Reply:
I agree that we should clarify whether the base64 encoding should be padded or not in this RFC, but we do need to identify which strategy we used in the original implementation and use that.  @dnault can you confirm that the 'MUST include padding' text reflects the existing implementations?

Reply:
For reference, the Node.js implementation does pad the base64, however based on the above-mentioned implementation of the signature generation, and the fact that the signature is used as a top-level construct, it shouldn't actually matter to the SDK if it is padded or not.  We should probably specify a "MUST include padding" or "MUST exclude padding" on the encode side, and a "MUST not care about padding" on the decode side.

Reply:
We do use padding.